history/characters/110000_scourge.txt

```diff
@@ -15,6 +15,7 @@
 		remove_trait=creature_orc
 		trait=creature_lich
 		trait=being_undead
+		effect = { add_trait_force_tooltip = creature_lich }
```

Comment:
There's no need to keep `trait=` line around.
P.S. The same with other `trait` that were replaced by `add_trait_force_tooltip`.
```suggestion
		trait=being_undead
		effect = { add_trait_force_tooltip = creature_lich }
```

Reply:
I cleaned up the code a bit based on your sugestion.